python/servo/bootstrap_commands.py

```diff
@@ -63,6 +63,9 @@ def download(desc, src, writer, start_byte=0):
 
         if not dumb:
             print()
+    except urllib2.URLError:
+        print("Error downloading Rust compiler; are you connected to the internet?")
```

Comment:
Nit: `Error downloading the Rust compiler. Please make sure that you're connected to the internet.`